packages/@okta/i18n/src/properties/login.properties

```diff
@@ -640,6 +640,15 @@ socialauth.google.label=Sign in with Google
 socialauth.linkedin.label=Sign in with LinkedIn
 socialauth.microsoft.label=Sign in with Microsoft
 socialauth.apple.label=Sign in with Apple
+socialauth.github.label=Sign in with GitHub
+socialauth.gitlab.label=Sign in with GitLab
+socialauth.yahoo.label=Sign in with Yahoo
+socialauth.line.label=Sign in with LINE
+socialauth.paypal.label=Sign in with PayPal
+socialauth.paypal_sandbox.label=Sign in with PayPal Sandbox
+socialauth.salesforce.label=Sign in with Salesforce
+socialauth.amazon.label=Sign in with Amazon
+socialauth.yahoo_japan.label=Sign in with Yahoo Japan
```

Comment:
Since this extends support for many more IdPs, can we update our [Social Login section](https://github.com/okta/okta-signin-widget/tree/2babf07e346349706dbc91d8ff9011e54db4e807#openid-connect) of the README to either link to or explicitly call out all supported IdPs?